### ISSUE TITLE
linter error being shown when it shouldn't

### ISSUE THREAD

**Issue by tylerthehaas** [PolymerLabs/polymer-atom#11]
getting <iron-ajax> is undefined in my editor.  I have verified the path to iron-ajax under bower_components is correct

``` html
<link rel="import" href="../../polymer/polymer.html">
<link rel="import" href="../ep-select-by-participant.html">
<link rel="import" href="../../iron-ajax/iron-ajax.html">

<style is="custom-style" include="demo-pages-shared-styles">
  .vertical-section-container {
    max-width: 960px;
  }
</style>

<dom-module is="demo-component">
  <template>
    <iron-ajax id="participants"
      auto
      url="http://localhost:3000/participants"
      handle-as="json"
      on-response="getItems"
      last-response="{{ajaxResponse}}"></iron-ajax>
```

**Comment by cschnaith** [comment]
I get the same issue. Atom-polymer says undefined for the import px-alert-label in the editor, but polylint runs without error.

```
<link rel="import" href="../px-alert-label/px-alert-label.html" />
...
<px-alert-label class="u-mr zeta" type$="{{alertLabelTypeIs(alert)}}"
                          label$="{{alertLabelIs(alert)}}">
</px-alert-label>

```
